<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ReqEncoderDlg</class>
 <widget class="QDialog" name="ReqEncoderDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Request Encoder</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="3">
    <widget class="QPushButton" name="mFindInputBtn">
     <property name="text">
      <string>FindInput</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>UUID</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="4">
    <widget class="QPlainTextEdit" name="mOutputText"/>
   </item>
   <item row="3" column="3">
    <widget class="QPushButton" name="mChangeBtn">
     <property name="text">
      <string>Change</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="mUUIDText"/>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Input</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Output</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="4">
    <widget class="QPlainTextEdit" name="mInputText"/>
   </item>
   <item row="5" column="0" colspan="4">
    <layout class="QGridLayout" name="gridLayout_2">
     <property name="topMargin">
      <number>0</number>
     </property>
     <item row="0" column="0">
      <widget class="QPushButton" name="mCreateBtn">
       <property name="text">
        <string>Create</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QPushButton" name="mGetBtn">
       <property name="text">
        <string>Get</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QPushButton" name="mActivateBtn">
       <property name="text">
        <string>Activate</string>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QPushButton" name="mDestroyBtn">
       <property name="text">
        <string>Destroy</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="6" column="0" colspan="4">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="topMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QPushButton" name="mEncryptBtn">
       <property name="text">
        <string>Encrypt</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="mDecryptBtn">
       <property name="text">
        <string>Decrypt</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="mSignBtn">
       <property name="text">
        <string>Sign</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="mVerifyBtn">
       <property name="text">
        <string>Verify</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="7" column="0" colspan="4">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <property name="topMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QPushButton" name="mRegisterBtn">
       <property name="text">
        <string>Register</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="mCreateKeyPairBtn">
       <property name="text">
        <string>CreateKeyPair</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="mSetAttributeBtn">
       <property name="text">
        <string>SetAttribute</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="mGetAttribute">
       <property name="text">
        <string>GetAttribute</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
